check/fixes.frm

```diff
@@ -2655,3 +2655,23 @@ assert stdout =~ exact_pattern(<<'EOF')
    nonempty(14): x y
 EOF
 *--#] Issue486 :
+*--#[ Issue508 :
+Off statistics;
+cf f g;
+s N;
+on highfirst;
+l F =
+#do i=1,5000
++ f(`i')*(g(1+N/`i')*g(N/`i'-1))^25
+#enddo
+;
+.sort
+makeinteger g;
+id f(N?) = N^50;
+id g(N?) = 1;
+print;
+.end
+#ulimit -v 8_000_000
```

Comment:
Though tests on Windows in CI are currently run with bash, I feel using `#require unix?` (or `linux?`) with `#ulimit` is reasonable; it might not be so natural to assume bash will always be available on Windows, especially if alternative build methods ([CMake](https://github.com/vermaseren/form/issues/467) etc.) are implemented in future.

I think you also need `#require wordsize == 4` because the amount of memory usage on 64-bit machines is assumed.